docs/integrations/nbext.md

```diff
@@ -1,47 +1,53 @@
 # Jupyter Notebook Extension
 
 Now you can commit your Jupyter Notebook with to [Jovian](https://jovian.ml?utm_source=docs) just **One Click**.
```

Comment:
There is a typo from a different commit, can you change it? `Jupyter Notebook to [Jovian](https://jovian.ml?utm_source=docs) with just **One Click**`

Reply:
Yeah